docs/concepts/fractions.adoc

```diff
@@ -0,0 +1,32 @@
+[#fractions]
+= Fractions
+
+WildFly Swarm is defined by an unbounded set of capabilities. 
+Each chunk of functionality is called a _fraction_.
+Some fractions directly align with commonly-used APIs, such as _JAX-RS_ or _CDI_.
+Other fractions exist to provided higher-order capabilities, such as integration with RHSSO (KeyCloak). 
```

Comment:
"to provided" -> "to provide"

"higher-order" sounds maybe too fancy?

"KeyCloak" -> "Keycloak" (not sure about RHSSO, maybe we should use the full form?)

Reply:
Addressed in bobmcwhirter/wildfly-swarm#2.